Source/interactivity_manager_impl.cpp

```diff
@@ -1020,6 +1020,26 @@ interactivity_manager_impl::trigger_cooldown(_In_ const string_t& control_id, _I
     }
 }
 
+void MICROSOFT_MIXER_NAMESPACE::interactivity_manager_impl::send_message(const string_t & message)
+{
+    // The goal of this API is to be fast, with minimal memory and CPU overhead. This is why
+    // it is a pass-through to the websocket rather than packaging the message into a data structure
+    // and putting it on a queue to be sent later.
+    m_webSocketConnection->send(message)
+        .then([](pplx::task<void> t)
+    {
+        try
+        {
+            t.get();
+        }
+        catch (std::exception e)
+        {
+            // Throws this exception on failure to send, our retry logic once the websocket comes back online will resend
+            LOGS_ERROR << "Failed to send on websocket.";

```

Comment:
Ignore - this will go away in my next iteration.

---
In reply to: [132345334](https://github.com/mixer/interactive-cpp/pull/25#discussion_r132345334) [](ancestors = 132345334,132345259,132343034)